core/adt/map.hpp

```diff
@@ -8,8 +8,6 @@
 #include "storage/hamt/hamt.hpp"
 
 namespace fc::adt {
-  using storage::hamt::Hamt;
-
```

Comment:
Why is it better to remove?

Reply:
it's used only once, and to remove from this namespace